azure-resources/Storage/storageAccounts/recommendations.yaml

```
@@ -40,15 +40,15 @@
     - name: Migrate your classic storage accounts to Azure Resource Manager
       url: "https://learn.microsoft.com/azure/storage/common/classic-account-migration-overview"
 
-- description: Ensure Performance tier is set as per workload
+- description: Use premium performance block blob storage
   aprlGuid: 5587ef77-7a05-a74d-9c6e-449547a12f27
   recommendationTypeId: null
   recommendationControl: Scalability
   recommendationImpact: Medium
   recommendationResourceType: Microsoft.Storage/storageAccounts
   recommendationMetadataState: Active
   longDescription: |
-    Consider using the appropriate storage performance tier for workload scenarios. Each workload scenario requires appropriate performance tiers, and selecting the appropriate tiers based on storage usage is crucial.
+    One or more Storage Accounts has a high transaction rate per GB of block blob data stored. Use premium performance block blob storage instead of standard performance storage for workloads that require fast storage response times and/or high transaction rates.
```

```suggestion
    Use premium performance block blob storage instead of standard performance storage for workloads that require fast storage response times and/or high transaction rates.
```

I think the first sentence is not needed as that should reside in the KQL description